src/main/js/Widgets/MultipleSelect.js

```diff
@@ -616,6 +623,17 @@ class MultipleSelect {
     return this;
   }
 
+  /**
+   * Sets selected options in the order that they were set to selected
+   *
+   * @param {string[]} selections The selected options.
+   * @returns {MultipleSelect} This.
+   */
+  withPreserveSelectionOrder(selections) {
```

Comment:
The method name and properly don't seem to match. I'll have to think about this a bit more... but this widget already knows what is selected because it will just look at which `options` have the `selected` attribute. 

So I guess what we need to tell the widget is that we have a particular order we want to use, and perhaps even a particular set of selected options. So is this config - just an override for selected options? And then it is up to the caller to send them in whatever order they wish?